docs/static/assets/demo.css

```diff
@@ -1,5 +1,8 @@
 body {
-    background-image: url("/img/geometry2.png")
+  background-image: url("/img/geometry2.png")
+}
+html.dark  body {
+  background: #41403e;
```

Comment:
I think putting this in a variable would be great!

Reply:
Yup, that's already the case in`$primary` variable, but here it's being used to style the docs in a .css file